lib/ronin/exploits/cli/commands/run.rb

```diff
@@ -85,10 +86,13 @@ class Run < ExploitCommand
           include Core::CLI::Options::Param
           include Core::CLI::Logging
           include CommandKit::Printing::Indent
+          include Support::CLI::Printing
 
           # Exploit options
-          option :dry_run, short: '-D',
-                           desc: 'Builds the exploit but does not launch it'
+          option :dry_run, desc: 'Builds the exploit but does not launch it'
```

Comment:
Lost the `short: '-D'` flag.

Reply:
I explained in the git commit -- the `-D` flag was being overridden by the `--debug`'s short flag. Tell me how you'd like to resolve.